Aztec/Classes/Libxml2/DOM/Data/TextNode.swift

```diff
@@ -250,7 +254,22 @@ extension Libxml2 {
             }
         }
 
+        func shouldSanitizeText() -> Bool {
```

Comment:
Can we add documentation to explain why this method is necessary?  While on topic it really makes sense, but if you grab this on its own, it may not be too obvious at first.

Also, for readability and code reusability it may be nice to change this method to:

```swift
func shouldSanitizeText() -> Bool {
    return hasParent(.pre)
}
```